dev-libs/libbsd/libbsd-0.11.7-r2.ebuild

```diff
@@ -34,6 +34,8 @@ src_prepare() {
 }
 
 multilib_src_configure() {
+	use elibc_musl && append-flags -D_LARGEFILE64_SOURCE
```

Comment:
This deserves a comment because it's a temporary workaround which won't last for very long (probably about a year at most until the next musl release).

It also, ideally, would be accompanied by a ref. to a Gentoo bug + any upstream references.

Reply:
Got it. Would a link to the musl changelog with the deprecation suffice?

Reply:
I'd prefer a specific in Gentoo, or, better, an upstream bug, because then it's very clear when we can get rid of it.

The danger otherwise is that we add a bunch of these, never report it upstream, then a year from now we have the same problem but with no escape hatch

Reply:
Gotcha. I'll file upstream bugs and link them if they don't already exist.

Reply:
Quick question, where would I do this for systemd-utils? Should it be added to the musl patchset?

Reply:
for systemd-utils, I think kraj has filed a PR somewhere. It's ok for you to link to that and pull it in and then whoever merges it can add it into the tarball.

I wouldn't report upstream for systemd though, just ensure there's a Gentoo bug